.github/workflows/precommit.yml

```diff
@@ -281,6 +281,7 @@ jobs:
         # Protobuf encoding differs per python version, use 3.8 by default.
         python-version: [ "3.8"]
         cloud-provider: [aws]
+        protobuf-version: ["3.20.1", "5.28.3"]
     steps:
```

Comment:
Might I suggest instead of testing it with every PR, we add it to daily test. This will allow us to cover 4.xx also